engine/engine.go

```diff
@@ -128,6 +130,10 @@ func (eng *Engine) Setup(ctx context.Context, specv runtime.Spec) error {
 		Files:     spec.Files,
 	}
 
+	if instance.Arch == ARM64 {
```

Comment:
can you put `const ArchARM64 = "arm64"` into oshelp similar to https://github.com/drone-runners/drone-runner-aws/pull/117/files#diff-2eb7297b270d1432e87ed95c084b3db7095e82eb638e3ae8e52e30af4f658774

Reply:
done